pkg/repository/buffer/bulk_events.go

```diff
@@ -27,6 +28,7 @@ type BulkEventWriter struct {
 
 func NewBulkEventWriter(pool *pgxpool.Pool, v validator.Validator, l *zerolog.Logger) (*BulkEventWriter, error) {
 	queries := dbsqlc.New()
+	flushPeriod := 250 * time.Millisecond
```

Comment:
@abelanger5 we didn't mention specifics when we chatted, 250 ms and 1000 items good here?